models/token.js

```diff
@@ -85,7 +83,9 @@ module.exports = {
      * @property update
      * @type {Joi}
      */
-    update: Joi.object(mutate(MODEL, [], [
+    update: Joi.object(mutate(MODEL, [
+        'id'
+    ], [
```

Comment:
This is intended to be used to validate the payload of the POST tokens/{id} right?
If you look at this: https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/jobs/update.js
`/{id}` is validated by validate param and the update schema is used to validate the payload. Do you want `id` inside the payload instead of part of the endpoint?